.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node: [18, 20.4]
+        node: [18, 20.4, lts/*, latest]
```

Comment:
```suggestion
        node: [18, 20, lts/*, latest]
```

Happy to give `latest` a go, but I haven't found that a great experience for library maintenance - with Pact, we got a lot of instability on the odd numbered versions, so we stopped building them.

I keep meaning to make a tool to keep these fields explicit and correct whenever node's release dates hit - along with a breaking change commit for dropping support. One day :)

Reply:
Makes sense, can Renovate keep these up to date?

Reply:
Not as far as I know, but surely something can..